CONTRIBUTING.md

```diff
@@ -131,6 +131,13 @@ There are a few important arguments:
 - `--slow-tests` enables running slow tests. See below for a description
   of slow tests.
 
+- `--with-cuda` sets the device fixture in [tests/influence/torch/conftest.py](
+  tests/influence/torch/conftest.py) to `cuda` if it is available.
+  Using this fixture within tests, you can run parts of your tests on a `cuda` 
+  device. Be aware, that you still have to take care of the usage of the device
```

Comment:
Maybe this can be a bit more explicit, eg say something like "Note that you need to use the fixture in your tests in order to benefit from this (...)"

Reply:
For now, I would leave it like that, until we get feedback, that this is not understandable.